dns/rfc2136/src/www/services_rfc2136_edit.php

```diff
@@ -149,14 +153,11 @@
                    <td><i class="fa fa-info-circle text-muted"></i> <?=gettext("Interface to monitor");?></td>
                    <td>
                      <select name="interface" class="selectpicker" id="requestif">
- <?php
-                      foreach (get_configured_interface_with_descr() as $if => $ifdesc):?>
+                        <?php foreach (get_configured_interface_with_descr() as $if => $ifdesc):?>
```

Comment:
If you want to improve style please kill the indent completely for the conditonals

Reply:
I believe this is fixed by https://github.com/opnsense/plugins/pull/3316/commits/3c829b658172f49c9266af289f97275305768b3a